x/poolmanager/router.go

```diff
@@ -4,6 +4,7 @@ import (
 	"errors"
 	"fmt"
 	"math/big"
+	"slices"
```

Comment:
Seems like CI is not happy with the new package, can you use Contains that we have made instead? 
https://github.com/osmosis-labs/osmosis/blob/main/osmoutils/slice_helper.go#L98

Reply:
Hi @mattverse , I have replaced it now